src/components/shared/formItems/ProjectTicketMods.tsx

```diff
@@ -272,7 +274,8 @@ export default function ProjectTicketMods({
             </span>
           </div>
           <div>
-            {100 - total}% to <FormattedAddress address={owner} />
+            {100 - total}% <Trans>to</Trans>{' '}
```

Comment:
**issue**: lets wrap whole thing in `Trans` to avoid splitting sentence.

Reply:
Thank you for your comments @aeolianeth! I've fixed up all those points.